### ISSUE TITLE
Passing multiple apidoc_module_dir's

### ISSUE THREAD

**Issue by rahasurana** [sphinx-contrib/apidoc#12]
We have a centralized doc generation where **sphinx-apidoc** is called on each repo and the generated rst files are used to generate the documentation,

Given this is a pre-stage operation done outside sphinx, current extension seemed like a good way to avoid the pre-stage creation of rst.
Only caveat here is that **apidoc_module_dir** only accepts a single path. Can this be enhanced to accept a  list of paths?

**Comment by M-Kerr** [comment]
Agreed. I think this is weakness of sphinx-apidoc in general, as it must be run once for _every_ source directory.
